src/Standards/Squiz/Sniffs/Commenting/FunctionCommentThrowTagSniff.php

```diff
@@ -51,16 +57,18 @@ protected function processTokenWithinScope(File $phpcsFile, $stackPtr, $currScop
         $find[] = T_WHITESPACE;
 
         $commentEnd = $phpcsFile->findPrevious($find, ($currScope - 1), null, true);
-        if ($tokens[$commentEnd]['code'] === T_COMMENT) {
-            // Function is using the wrong type of comment.
-            return;
-        }
+        if ($this->skipFunctionsWithoutDocComment === true) {
```

Comment:
These changes are looking a bit confusing, but there is only added that condition and for the body is added one more indent.